### PR TITLE
AC-613: Change file structure to match play-publisher's conventions

### PR DESCRIPTION
**Pull request by deepak140596**
## Description of what I changed

Structured the files according to [this format](https://github.com/Triple-T/gradle-play-publisher#uploading-release-notes).
Changed camelCase format to kebab-case for files as per latest instructions.
Added release notes to Readme.md file

## Issue I worked on

JIRA Issue: https://issues.openmrs.org/browse/AC-613

## Checklist: I completed these to help reviewers :)

- [x] My pull request only contains **ONE single commit**
(the number above, next to the 'Commits' tab is 1).

- [ ] I have **added tests** to cover my changes. (If you refactored
existing code that was well tested you do not have to add tests)

- [x] All new and existing **tests passed**.

- [x] My pull request is **based on the latest changes** of the master branch.

### CODE REVIEW

File: openmrs-client/build.gradle [hunk 1]
```diff
@@ -177,4 +177,8 @@ play {
     }
 
     serviceAccountCredentials = rootProject.file('google_play.json')
+    resolutionStrategy = "auto"
```
Comment (f4ww4z) [inline openmrs-client/build.gradle]: @deepak140596 What is the function for this?

File: openmrs-client/build.gradle [hunk 2]
```diff
@@ -177,4 +177,8 @@ play {
...
+    outputProcessor { // this: ApkVariantOutput
+        versionNameOverride = "$versionNameOverride.$versionCode"
+    }
```
Comment (f4ww4z) [inline openmrs-client/build.gradle]: Add the `releaseStatus` attribute.

File: openmrs-client/build.gradle (hunk 1 above)
Comment (deepak140596) [reply openmrs-client/build.gradle]: This helps us in
"Automatically pick the correct version code so you don't have to manually update it (auto)"

File: openmrs-client/build.gradle (hunk 2 above)
Comment (deepak140596) [reply openmrs-client/build.gradle]: I will make the necessary changes

**Pull request merged**